tests/pytests/unit/modules/test_event.py

```diff
@@ -0,0 +1,103 @@
+"""
+    :codeauthor: Rupesh Tare <rupesht@saltstack.com>
+
+    Test cases for salt.modules.event
+"""
+
+import pytest
+
+import salt.modules.event as event
+import salt.utils.event
+from tests.support.mock import MagicMock, patch
+from tests.support.paths import SALT_CODE_DIR
+
+
+@pytest.fixture
+def configure_loader_modules():
```

Comment:
I believe we now have a `minion_opts` fixture available. Let's use that instead.
Ping me for additional clarifications if need be.

Reply:
Hopefully did this right, thanks.